colcon_core/location.py

```diff
@@ -109,12 +109,16 @@ def set_default_log_path(*, base_path, env_var=None, subdirectory=None):
 _create_log_path_lock = Lock()
 
 
-def create_log_path():
+def create_log_path(verb_name):
     """
     Create the logging directory if it doesn't exist yet.
 
-    On non-Windows platforms a symlink named `latest` is created in the base
-    path which links to the subdirectory.
+    On non-Windows platforms two symlinks are created as siblings of the log
+    path:
+    * `latest_<verb_name>` linking to the log path
+    * `latest` linking to `latest_<verb_name>`
+
+    :param str verb_name: The verb name
```

Comment:
Question: This seem inconsistent across the project (even within this file), should the parameter type be specified within the `:param` tag or not?

It seems common to specify the type on another line using the `:type` tag

Reply:
This file looks consistent to me. I assume you refer the use of `:rtype:`? For parameters the type can be mentioned between the keyword and the parameter name. I don't think that is possible for return values.

Reply:
I was refering to https://github.com/colcon/colcon-core/blob/584206d2ef316739e5f34d7279fc9efe1592f812/colcon_core/location.py#L49 for example

Reply:
In that case the argument is not a "simple" builtin type but an instance of a specific class. I often don't put the whole type name in there since it makes the part super long and I haven't experimented with the API extraction yet.

Reply:
both 
```
:param str verb_name: The verb name
```
and
```
:param verb_name: The verb name
:type verb_name: str
```
are valid in sphinx, it's just a matter of what convention we chose to follow.

Reply:
Sure, as I mentioned almost all of the code uses the first option to avoid the duplication.

Reply:
Sure, as I mentioned almost all of the code uses the first option to avoid the duplication. But not all cases specify the type if it is not a "primitive" type.

Reply:
sounds good :+1: